src/panels/lovelace/cards/hui-statistics-graph-card.ts

```diff
@@ -167,12 +172,21 @@ export class HuiStatisticsGraphCard extends LitElement implements LovelaceCard {
         1000 * 60 * 60 * (24 * (this._config!.days_to_show || 30) + 1)
     );
     try {
+      const metadata = await getStatisticMetadata(this.hass!, this._entities);
+      const unitClass = metadata?.[0]?.unit_class;
+      const displayUnit = getDisplayUnit(
+        this.hass!,
+        metadata?.[0]?.statistic_id,
+        metadata?.[0]
+      );
```

Comment:
Picking the 0th element here is not correct because the order of the response list is not same as the order in the request.
Should we modify `getStatisticMetadata` to return a map, indexed by statistic_id, instead of a list?